task_01.rst

```diff
@@ -0,0 +1,103 @@
+############
+Why I Love Python
+############
```

Comment:
Try to make your header over/underline the same length as your text, eg, if your header text is 10 characters long, (spaces included) make the header the same:

```
##############
Apple Smoothie
##############
```